changelogs/fragments/replace-random-with-secrets.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - replace random.SystemRandom() with secrets.SystemRandom() when
+    generating passwords and issue a warning if a seed is provided
```

```suggestion
  - password lookup plugin - replace random.SystemRandom() with secrets.SystemRandom() when
    generating passwords and issue a warning if a seed is provided (https://github.com/ansible/ansible/issues/85956, https://github.com/ansible/ansible/pull/85971).
```
Also note that your PR does **not** issue a warning. You only adjust documentation to contain a warning. If you don't plan to do the former, you should remove that part from the changelog fragment.

---

yeah, that warning will be in the second push to this PR